### PR TITLE
feat: added logger

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**

### CODE REVIEW

File: log/logger.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                            filename=self.__config['logger']['filename'],
+                            datefmt=self.__config['logger']['dateFormat'],
+                            format=self.__config['logger']['format'])
+        self.log = logging.getLogger(__name__)
+        self._create_file_handler()
+
+    def _create_file_handler(self):
+        if self.log.handlers:
+            self.log.handlers = []
+        file_handle = logging.FileHandler(r'F:/Repos/exporter-worker/src/ew_logs.log')
```
Comment (asafMasa) [inline log/logger.py]: hard coded - get from config
add log files to git ignore
Why don't you use winston logger? or some other infra logger?

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: log/logger.py (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply log/logger.py]: -Why don't you use winston logger? or some other infra logger? 
A: Python already have 'Logger' module built-in called 'logging' same as winston , very easy to use.

Comment (CL-SHLOMIKONCHA) [reply log/logger.py]: fixed in c42b798

File: log/logger.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import logging
+from os import path
+import json
+from sys import stdout
+
+
+class Logger:
+    def __init__(self):
+        current_dir_path = path.dirname(__file__)
+        config_path = path.join(current_dir_path, '../confd/config/default.json')
```
Comment (asafMasa) [inline log/logger.py:10]: add an issue to move the config file reader to one class that will be imported by all who needs it

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: log/logger.py (hunk 2 above)
Comment (CL-SHLOMIKONCHA) [reply log/logger.py:10]: issue created

**Review by asafMasa — APPROVED** [review]

**Pull request merged**